nucleus/utils.py

```diff
@@ -187,7 +188,7 @@ def format_dataset_item_response(response: dict) -> dict:
     }
 
 
-def convert_export_payload(api_payload):
+def convert_export_payload(api_payload, model_id: str = None):
```

Comment:
Would make more sense to pass a boolean instead of nullable string here. Since we're not using the
value of `model_id` anywhere.

ie: `convert_export_payload(api_payload, has_predictions: bool = False):`